opennms-doc/guide-admin/src/asciidoc/text/telemetryd/troubleshooting.adoc

```diff
@@ -0,0 +1,57 @@
+
+[[telemetryd-troubleshooting]]
+=== Troubleshooting
+
+Configuring _telemetryd_ involves a few components.
+This section gives a few troubleshooting hints to verify if the configuration and the network setup is correct.
+
+.Ensure the Java process is listening on a specific port
+[source]
+----
+ss -lnp sport = :8877
```

Comment:
Why port 8877?

Reply:
It's an unregistered port, just as an example. Any better idea?

Reply:
If we're talking about telemetryd, we should probably tie this to a specific protocol i.e.:

For example, the default UDP listener for JTI listens on UDP port 50000. To verify if the Java process is listening on this port...